data/__generated__/schema.graphql

```diff
@@ -164,6 +164,30 @@ input ImageCreateInput {
   metadata: JSON
 }
 
+input ImageCreateManyInput {
+  images: [ImageCreateManySingleImage!]!
+  datasetId: ID!
+}
+
+input ImageCreateManySingleImage {
+  id: ID
```

Comment:
Why would we allow to force an ID property?

Reply:
It's sometimes useful to generate id on the client side. For example, with optimistic responses, or for the undo/redo